Create Openings Page to list positions in the Team that need to be filled. (#34)
QuiltMC/quiltmc.org: src/_community/openings.md

---
title: Openings
description: Current Openings for positions on the Quilt Staff Team

redirect_from:
  - /community/openings.html

> ---

Reviewer: Redirect is unnecessary. Also, please stick a newline after the front matter.

> The Quilt Project is very community-oriented, and that means that it needs your help to succeed. This page is a list of all the places we currently need help with, both in development and in our community.

Reviewer: "All the places"?

Author: The language may need work

Reviewer: I think the original wording was fine, you could've just changed it to "this page" and it'd work imo.

{% admonition %}

> **Please note:** The listed positions typically do not have a quota. As long as a position remains listed in this channel, we'll continue to be interested in reviewing and accepting more members for them.

Reviewer 2: ```suggestion
**Please note:** The listed positions typically do not have a quota. As long as a position remains listed on this page, we'll continue to be interested in reviewing and accepting more members for them.
```

> **Please note:** The listed positions typically do not have a quota. As long as a position remains listed in this channel, we'll continue to be interested in reviewing and accepting more members for them.

Reviewer: Not a channel.

{% endadmonition %}

## Development Roles

Development teams build the tools that make up Quilt, from build tools, to mappings, to libraries, to loaders. Joining a development team is a great way to contribute if you have a knowledge of Java, and how Minecraft works behind the scenes.

### Build Tools

The Build Tools team is responsible for building the tools that make Quilt modding possible. This is one of our most important teams, and it maintains projects that are instrumental for getting Quilt to a full release.

This team is currently focused on improving Sponge's VanillaGradle and bringing it into a state where it can be used for Quilt. Until this is completed, Quilt projects are reliant on Fabric's Loom and a set of additional Gradle plugins, so we're hoping to prioritise development in this team.

> If you're interested in helping out with Quilt's build tools or would like to join this team, please send a message in #build-tools-general on the [Toolchain Discord server](https://discord.quiltmc.org/toolchain).

Reviewer: Did we ever figure out whether build tools wanted to be on the list?

Author: Yes, I asked Glitch and he said that it changes frequently but at the moment he did want to be on the list.

### Mappings Triage

The Mappings team is responsible for Quilt Mappings and its backbone projects, such as Hashed Mojmap. This team is currently looking for triage members, who are mostly responsible for reviewing Pull Requests to Quilt Mappings, ensuring they're up to spec and meet the project's requirements.

This isn't a high-stress position, but it's still important - especially as we'd like to be able to allow Quilt Mappings PRs to move quickly and be merged without unnecessary delay.

> If you'd like to join this team, please send a message in #mappings-general on the [Toolchain server](https://discord.quiltmc.org/toolchain).

Reviewer: For emphasis and readability, you may as well wrap channel names in backticks.

### Worldgen Developers

Both the Mappings and QSL teams are in need of developers experienced with Minecraft's world generation. While this is technically a specialist position, you don't need to be a definitive expert, just someone with experience in this area of modding.

> If this sounds like you and you'd like to help out, please send a message in #mappings-general or #qsl-general on the [Toolchain server](https://discord.quiltmc.org/toolchain).

Reviewer 2: Should be either `Toolchain Discord server` or `Toolchain server`, not both IMO.

Author: The idea is you read the full thing the first time and then know what "Toolchain server" means all subsequent times, but you could be right.

### QSL

There are several QSL teams that don't yet have any members, and this means that the parts of QSL that they are responsible for is overseen by the core QSL team instead, which isn't ideal as they already have a lot of work to do. The following teams are empty, and need members:

-  Block
-  GUI
-  Rendering
-  Worldgen

Additionally, the following QSL teams are looking for more members:

-  Entities

> If you'd like to join one of these QSL teams and/or work on these modules, please send a message in #qsl-general on the Toolchain server, which you can find a link to in toolchain-development.

Reviewer 2: ```suggestion
If you'd like to join one of these QSL teams and/or work on these modules, please send a message in #qsl-general on the [Toolchain Discord server](https://discord.quiltmc.org/toolchain).
```
